android/app/src/main/java/com/zulipmobile/notifications/NotificationHelper.java

```diff
@@ -65,22 +66,22 @@ public static String extractName(String key) {
         return key.split(":")[0];
     }
 
-    public static void buildNotificationContent(LinkedHashMap<String, Pair<String, Integer>> conversations, Notification.InboxStyle inboxStyle, Context mContext) {
-        for (Map.Entry<String, Pair<String, Integer>> map : conversations.entrySet()) {
+    public static void buildNotificationContent(LinkedHashMap<String, List<MessageInfo>> conversations, Notification.InboxStyle inboxStyle, Context mContext) {
+        for (Map.Entry<String, List<MessageInfo>> map : conversations.entrySet()) {
```

Comment:
(This is for a followup, because it's pre-existing)
This variable is not a map -- it's an entry in a map. `entry` would be a good name.

Similarly below.